tools/loadtesting/terraform/main.tf

```diff
@@ -0,0 +1,86 @@
+variable "region" {
+  default = "us-east-2"
+}
+
+provider "aws" {
+  region = var.region
+  default_tags {
+    tags = {
+      environment = "loadtest"
+      terraform   = "https://github.com/fleetdm/fleet/tree/main/tools/terraform"
+      state       = "local"
+    }
+  }
+}
+
+terraform {
+  // these values are hard-coded to prevent chicken before the egg situations
+  #  backend "s3" {
+  #    bucket = "fleet-terraform-remote-state"
+  #    region = "us-east-2"
+  #    key = "fleet/"
+  #    dynamodb_table = "fleet-terraform-state-lock"
+  #  }
```

Comment:
Do we still need this commented out or can we remove it?

Reply:
Anything commented can be removed